readme.md

```diff
@@ -622,6 +624,8 @@ interface RequestCredential {
             requiredProperties: string[]
         }
     ]
+    /** Target DID URI*/
+    targetDid: string
```

Comment:
We called this `owner` every where else. Would it make sense to also call it owner here? It is a `RequestCredential` message. So you could argue that the owner field is the owner of the credential that is requested. `subject` would maybe be even more precise, but inconsistent with the rest of the API.

Reply:
According to the spec the message can be sent both ways, but in our use cases its always the dapp.